Problem1_Queue using stacks.py

```diff
@@ -0,0 +1,49 @@
+class MyQueue(object):
+
+    def __init__(self):
+        """
+        Initialize your data structure here.
+        """
+        self.s1 = []
+        self.s2 = []
+        
+
+    def push(self, x):
+        """
+        Push element x to the back of queue.
+        :type x: int
+        :rtype: None
+        """
+        return self.s1.append(x)
+
+    def pop(self):
+        """
+        Removes the element from in front of queue and returns that element.
+        :rtype: int
+        """
+        return self.s1.pop(0)
+
+    def peek(self):
+        """
+        Get the front element.
+        :rtype: int
+        """
+        return self.s1[0]
+        
```

Comment:
We can use the stack functions of push, pop, peek only `return self.s1[0]` should not be used